openmetadata-ui/src/main/resources/ui/playwright/utils/customizeLandingPage.ts

```diff
@@ -117,3 +118,37 @@ export const setUserDefaultPersona = async (
     page.locator('[data-testid="user-profile-details"]')
   ).toContainText(personaName);
 };
+
+export const openAddWidgetModal = async (page: Page) => {
+  const fetchResponse = page.waitForResponse(
+    '/api/v1/docStore?fqnPrefix=KnowledgePanel*'
+  );
+  await page
+    .locator(
+      '[data-testid="ExtraWidget.EmptyWidgetPlaceholder"] [data-testid="add-widget-button"]'
+    )
+    .click();
+
+  await fetchResponse;
+};
+
+export const navigateToLandingPage = async (page: Page) => {
+  const feedResponse = page.waitForResponse(`/api/v1/feed*`);
+  const dataInsightResponse = page.waitForResponse(
+    `/api/v1/analytics/dataInsights/system/charts/name/total_data_assets/data?*`
+  );
+
+  // Click on the logo to navigate to the landing page
+  await page.locator('#openmetadata_logo').click();
+
+  await feedResponse;
+  await dataInsightResponse;
+};
```

Comment:
We already have redirect to home page function, we can utilize that instead of a new function